cumulusci/tasks/release_notes/tests/test_task.py

```diff
@@ -1,15 +1,27 @@
 from unittest import mock
 import pytest
+import os.path
 
 from github3.pulls import ShortPullRequest
 
 from cumulusci.core.config import TaskConfig
 from cumulusci.core.config import ServiceConfig
 from cumulusci.tests.util import create_project_config
-from cumulusci.tasks.release_notes.task import GithubReleaseNotes
+from cumulusci.tasks.release_notes.task import GithubReleaseNotes, AllGithubReleaseNotes
 from cumulusci.tasks.release_notes.task import ParentPullRequestNotes
 from cumulusci.tasks.github.tests.util_github_api import GithubApiTestMixin
 
+from cumulusci.tasks.salesforce.tests.util import create_task
+
+class TestAllGithubReleaseNotes:
+    def test_run_AllGithubReleaseNotes_task(
+        self,
+    ):
+        task = create_task(AllGithubReleaseNotes, options={"repos": [{"owner": "SalesforceFoundation", "repo": "NPSP"}]})
+        task.github = mock.Mock()
+        task.get_repo = mock.Mock()
+        task._run_task()
+        assert os.path.isfile("./github_release_notes.html")
```

Comment:
let's stick with using pathlib for assertions like this. You can `from pathlib import Path` and then this should be `assert Path('./github_release_notes.html').is_file()`.

Reply:
I'd also like to see some level of assertion that the task is behaving as expected, i.e., attempting to include each of the named repositories. We probably want to run this in a tempdir as well so that the CWD is not polluted.

Reply:
I think we should also be validating the output of the html file to a certain extent. You should be able to mock a response from github where we are getting back the change notes from a repository, and then assert that the actual file contents are what is expected given the mocked change notes.